core-primitives/top-pool-author/src/api.rs

```diff
@@ -52,6 +54,14 @@ impl<Block> SidechainApi<Block> {
 	pub fn new() -> Self {
 		SidechainApi { _marker: Default::default() }
 	}
+
+	fn validate_trusted_call(trusted_call_signed: TrustedCallSigned) -> ValidTransaction {
+		let from = trusted_call_signed.call.account();
+		let requires = vec![];
+		let provides = vec![from.encode()];
```

Comment:
I think @masapr fixed this, but only on a local branch.. but since you're touching this any way, may I ask you to change this to the following: `let provides = vec![(from, trusted_call_signed.nonce).encode()];` Otherwise, the top pool will only allow one call per account to reside inside it.

Reply:
Yes, I can do that 👍